### ISSUE TITLE
Incorrect typings for `smembers`

### ISSUE THREAD

**Issue by RRamirez1414** [upstash/upstash-redis#280]
```javascript
const redis = Redis.fromEnv()

await redis.sadd("key", {a: 1, b: 2}) 
const result = await redis.smembers("key")

console.log(typeof result[0]) // type is object
```

Hi! We're using upstash/redis v1.18.1 in our node environment. We noticed `smembers` returns `Promise<string[]>` but ends up returning non-stringified values when calling `sadd` to store objects

We're thinking the return value should be `Promise<TData[]>`

**Comment by chronark** [comment]
@RRamirez1414 yeah good catch, thank you
I'll fix it tomorrow

**Comment by chronark** [comment]
Fixed it in `v1.18.3`
![image](https://user-images.githubusercontent.com/18246773/208427384-b197dd19-2c96-4892-aefc-2913d4e29387.png)
